Source/Projections/Builder/Copies/MongoDB/ConversionsFromBsonClassMapBuilder.cs

```diff
@@ -11,8 +12,19 @@ namespace Dolittle.SDK.Projections.Builder.Copies.MongoDB;
 /// </summary>
 public class ConversionsFromBsonClassMapBuilder : IBuildPropertyConversionsFromBsonClassMap
 {
+    readonly IAddConversionsFromBsonClassMap _conversionsFromBsonClassMap;
+    
+    /// <summary>
+    /// Initializes a new instance of <see cref="ConversionsFromBsonClassMapBuilder"/> class.
+    /// </summary>
+    /// <param name="conversionsFromBsonClassMap">The <see cref="IAddConversionsFromBsonClassMap" />.</param>
+    public ConversionsFromBsonClassMapBuilder(IAddConversionsFromBsonClassMap conversionsFromBsonClassMap)
+    {
+        _conversionsFromBsonClassMap = conversionsFromBsonClassMap;
+    }
+    
     /// <inheritdoc />
-    public bool TryBuildFrom<TReadModel>(IClientBuildResults buildResults, IPropertyConversions conversions)
+    public void BuildFrom<TReadModel>(IClientBuildResults buildResults, IPropertyConversions conversions)
```

Comment:
Not really an ask for a change, but just wondering in terms of performance. In this case, we could get away with the method taking in a `Type` as a parameter. But when doing it with generics like this, doesn't this mean that there will be one method generated for every type behind the scenes? If so, this might not be a best practice to keep using?

Reply:
It seems like there is some overhead https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/generics/generics-in-the-run-time :)

Reply:
Let's not overdo it with Generics, but use it where it makes sense